src/SFA.DAS.ApprenticePortal/Models/ApprenticeHomepage.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.ApprenticePortal.Models
+{
+    public class ApprenticeHomepage
+    {
+        public Apprentice apprentice { get; set; }
+        public Apprenticeship apprenticeship { get; set; }
+    }
```

Comment:
I definitely think the names should be have the first letter as a capital. 

The model might more sense as

public class ApprenticeHomepage
    {
        public ApprenticeId { get; set;
        public ApprenticePersonalDetails ApprenticePersonalDetails { get; set; }
        public Apprenticeship LatestApprenticeship { get; set; }
    }

Reply:
its been given the same name as the object it represents - unnecessarily changing names just adds confusion